pl.json

```diff
@@ -345,6 +345,7 @@
         "Account.ErrorEmailMismatch" : "Emaile nie są takie same!",
         "Account.ErrorLoginError" : "Błąd Logowania:",
         "Account.RegistrationSuccess" : "Udało się! Został wysłany email z linkiem potwierdzającym. Po otworzeniu linku będziesz w stanie się zalogować.",
+        "": "<size=120%>Udało się!</size><br><br>Email z linkiem potwierdzającym został wysłany do {email}.<br><br>Po otworzeniu linku, zostaniesz zalogowany automatycznie jeżeli to okienko jest otwarte we'll automatically log you in as long as this dialog is open.<br><br>Jeśli napotkasz problemy, napisz do nas na hello@neos.com lub discord.gg/neosvr",
```

Comment:
what is this

Reply:
Looks like I copied the English value into an empty key to have this and previous version side-by-side (English version seems to have been updated), but didn't fix the key afterwards. Thanks for catching this